tests/test_docker.py

```diff
@@ -22,11 +22,17 @@ def add_numbers(in_1: float, in_2: float) -> float:
     output = graph.run(3.0, 4.0)
     assert output[0] == 3 + 4
 
-    docker.create_pipeline_api([graph], output_dir="./")
+    test_dir = "./docker_test"
+    docker.create_pipeline_api([graph], output_dir=test_dir)
 
     # Check for the creation of docker files
-    assert os.path.exists("./Dockerfile") and os.path.exists("./docker-compose.yml")
+    assert os.path.exists(f"{test_dir}/Dockerfile") and os.path.exists(
+        f"{test_dir}/docker-compose.yml"
+    )
     # Check for the serialization of the graph
-    assert os.path.exists("./AddNumbers.graph")
+    assert os.path.exists(f"{test_dir}/AddNumbers.graph")
+
+    if os.path.exists(test_dir):
+        shutil.rmtree(test_dir)
```

Comment:
Also nbd but you could leverage [pytest's existing tmpdir support](https://docs.pytest.org/en/6.2.x/tmpdir.html) to avoid having to do the cleanup by hand.

Reply:
aye aye hahah